src/core/components/RootNavLayout.css

```diff
@@ -0,0 +1,39 @@
+.root-layout-container {
+  position: fixed;
+  overflow: auto;
+  width: 100%;
+  display: grid;
+  grid-template-rows: 7.5vh 84vh 8.5vh;
+  height: 100%;
```

Comment:
We dont need to use height if the position is fixed and it is set with top and bottom values. Top 0 and bottom 0  are equal to 100vh

Reply:
Got it! Fixed!